alexandria/settings/alexandria.py

```diff
@@ -190,6 +190,15 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
     "ALEXANDRIA_DAV_SCHEME",
     default=default("ms-word:ofe|u|http", "ms-word:ofe|u|https"),
 )
+ALEXANDRIA_MANABI_ALLOWED_MIMETYPES = env.list(
+    "ALEXANDRIA_MANABI_ALLOWED_MIMETYPES",
+    default=[
+        "application/msword",
+        "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+        "application/vnd.ms-excel",
+        "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
```

Comment:
```suggestion
        "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
        "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
```
Let's not enable `.doc` and `.xls` files per default